composer.json

```diff
@@ -28,10 +28,11 @@
     "require-dev": {
         "squizlabs/php_codesniffer": "^2.8",
         "phpmd/phpmd": "^2.6",
-        "phpunit/phpunit": "^6.1"
+        "phpunit/phpunit": "^6.1",
+        "php-ds/php-ds": "^1.1"
```

Comment:
I put `php-ds/php-ds` in `suggest` section because I thought that `ext-ds` will be installed but you are right, `php-ds/php-ds` is required and `ext-ds` is optionally.

Reply:
and it should be in `require` section instead of `require-dev`?

Reply:
ah yes thats right